components/infobox/wikis/stormgate/infobox_unit_custom.lua

```diff
@@ -76,36 +76,23 @@ function CustomInjector:parse(id, widgets)
 		}
 	elseif id == 'cost' then
 		return {
-			Cell{
-				name = 'Cost',
-				content = {
-					args.informationType ~= 'Hero' and CostDisplay.run{
-						faction = caller.faction,
-						luminite = args.luminite,
-						luminiteTotal = args.totalluminite,
-						luminiteForced = true,
-						therium = args.therium,
-						theriumTotal = args.totaltherium,
-						theriumForced = true,
-						supply = args.supply,
-						supplyTotal = args.totalsupply,
-						supplyForced = true,
-						animus = args.animus,
-						animusTotal = args.totalanimus,
-					} or nil
-				}
-			},
-			Cell{
-				name =
-					args.buildtime and args.charge_time and 'Built' .. HOTKEY_SEPERATOR .. 'Recharge Time' or
-					args.charge_time and 'Recharge Time' or
-					'Built Time',
-				content = {
-					args.buildtime and args.charge_time and args.buildtime .. 's' .. HOTKEY_SEPERATOR .. args.charge_time .. 's' or
-					args.charge_time and args.charge_time .. 's' or
-					args.buildtime and args.buildtime .. 's' or nil
-				}
-			},
+			Cell{name = 'Cost', content = {args.informationType ~= 'Hero' and CostDisplay.run{
```

Comment:
question to check on discord: do we want to display costs (with 0's) for heros or do we want costs cell for them not to show at all?

Reply:
I guess with Adico on the side of not showing costs for heroes, we can resolve this conversation and finalize this pr.